core/metadata/src/impl/LoadUnitMultiInstanceProtoHook.ts

```
@@ -1,25 +1,25 @@
       }
     }
-    for (const clazz of multiInstanceClazzList) {
-      const protos = await EggPrototypeCreatorFactory.createProto(clazz, loadUnit);
-      for (const proto of protos) {
-        EggPrototypeFactory.instance.registerPrototype(proto, loadUnit);
-      }
-    }
+  }
+
+  static clear() {
+    this.multiInstanceClazzSet.clear();
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)